### ISSUE TITLE
v3.0.0 - PropTypes warning

### ISSUE THREAD

**Issue by ipanasenko** [react-grid-layout/react-draggable#276]
Hey guys. I've updated to v3.0.0 and now I see a warning in console:

```
Warning: Accessing PropTypes via the main React package is deprecated, and will be removed in  React v16.0. Use the latest available v15.* prop-types package from npm instead. For info on usage, compatibility, migration and more, see https://fb.me/prop-types-docs
```

Didn't check, but I think this change in a couple places is to blame: https://github.com/mzabriskie/react-draggable/commit/e405109dea195e12af7f3c98e646617655b0cf1a#diff-f5848504b1e287185730035e153a214fR2

Can you check, please?

What's the reason behind such change?

**Comment by STRML** [comment]
>  in a couple places 

Which places?

We are not using `React.PropTypes` in this module, which is easy to verify.

**Comment by ipanasenko** [comment]
I'm specifically updating react-draggble from 2.2.6 to 3.0.0 and I start seeing PropTypes warnings. I'm trying to find out what causes it. My thought is `import * as React from 'react';` somehow triggers using of PropTypes.

**Comment by mixtur** [comment]
Same with me.
Looks like when babel sees `import * as React from 'react'` it generates code that iterates over all exports of React, therefore PropTypes is also touched, which in turn triggers warning.
The problem should probably go away with upcoming React update, but for now it is very annoying.

**Comment by mixtur** [comment]
there it is: https://github.com/mzabriskie/react-draggable/blob/master/dist/react-draggable.js#L758, https://github.com/mzabriskie/react-draggable/blob/master/dist/react-draggable.js#L780

**Comment by STRML** [comment]
This is fixed in `3.0.3`.

**Comment by ipanasenko** [comment]
@STRML thank you!
